impl/src/test/java/io/kroxylicious/testing/kafka/invm/LoggingPrintStreamTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.testing.kafka.invm;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@ExtendWith(MockitoExtension.class)
+class LoggingPrintStreamTest {
+
+    @Mock
+    private System.Logger logger;
+
+    @Test
+    void singleLog() {
+        try (var lps = LoggingPrintStream.loggingPrintStream(logger, System.Logger.Level.INFO)) {
+            lps.println("Hello, world");
+
+            verify(logger, times(1)).log(System.Logger.Level.INFO, "{0}", "Hello, world");
```

Comment:
`times(1)` is implicit to `verify()` so arguably its just noise in the tests but they I wonder if it actually aids readability in context. The explicit call to times has me wondering if there should be confirming other numbers of invocations to.

Reply:
I'll use the default.  I'd forgotten times(1) was implict.